commands/dns_cli.py

```diff
@@ -448,8 +453,10 @@ def check_domain(domain_profile, project_profile, base_domain):
                             click.secho("Environment: " + env + "\t=> Domain: " + domain["http"]["alias"], fg="yellow")
                             cert_arn = check_r53(domain_session, project_session, domain["http"]["alias"], base_domain)
                             cert_list.update({domain["http"]["alias"]: cert_arn})
+                            click.echo(" ")
```

Comment:
What is this for?

Reply:
Makes a line break after each environment's block of output. Helps with grokking.

Reply:
It's going away now.